server/migrations/1616190016615_post-import-cleanup.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable camelcase */
+const { dbClient, collections } = require('../db');
+
+exports.up = async () => {
```

Comment:
Suggestion - could we add some basic logging to this just to let us know how many records it updated? We included that in our last few migrations and it proved super useful 🙌